<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NeurofeedbackBSettingsViewWidget</class>
 <widget class="QWidget" name="NeurofeedbackBSettingsViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="horizontalSpacing">
      <number>20</number>
     </property>
     <property name="verticalSpacing">
      <number>9</number>
     </property>
     <item row="3" column="3">
      <widget class="QLabel" name="label_StatusObject">
       <property name="text">
        <string>not loaded</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Max</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="pushButton_StdMin">
       <property name="text">
        <string>Standard</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_StatusBackground">
       <property name="text">
        <string>not loaded</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_Background">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButton_Object">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_DirBackground"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_DirObject"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Min</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_Object">
       <property name="text">
        <string>Object</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="pushButton_StdMax">
       <property name="text">
        <string>Standard</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_Background">
       <property name="text">
        <string>Background</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="spinBox_Max"/>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="spinBox_Min"/>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QPushButton" name="pushButton_ResetSettings">
       <property name="text">
        <string>Reset Settings</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
